src/generated/resources/data/forge/tags/fluids/default_fluid_physic.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why is this file here without a data generator? It'll be deleted the next time you run datagen.

Reply:
how can i gen it?